src/Avalonia.X11/X11Window.cs

```diff
@@ -1033,6 +1033,15 @@ public void SetParent(IWindowImpl? parent)
         public void Show(bool activate, bool isDialog)
         {
             _wasMappedAtLeastOnce = true;
+            var data = new List<nint>();
+            if (activate)
+            {
+                data.Add(_x11.Atoms.WM_TAKE_FOCUS);
+            }
+            
+            XChangeProperty(_x11.Display, _handle, _x11.Atoms.WM_PROTOCOLS, _x11.Atoms.XA_ATOM, 32,
```

Comment:
You're overwriting the existing protocols that we set ourselves on window creation:

https://github.com/AvaloniaUI/Avalonia/blob/353ad7dbd4d2b6e77b56a23a89c7615ac07f8389/src/Avalonia.X11/X11Window.cs#L226-L232

This is just wrong and will cause bugs, such as `Closing` not being raised or XSync not being used.
Plus, we don't even set `WM_TAKE_FOCUS` without `EnableInputFocusProxy` enabled in the first place.